### PR TITLE
help: add docs for periodic runs and builds

### PR DESCRIPTION
**Pull request by pearkes**

### CODE REVIEW

File: packer/builds/scheduling-builds.html.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Schedule Periodic Builds in Atlas
+
+<div class="alert-infos">
+  <div class="alert-info">
+    This is an unreleased beta feature. Please <a href="/help/support">contact support</a> if you are interested in helping us test this feature.
+  </div>
+</div>
+
+Atlas can optionally automatically run a Packer build and
```
Comment (sethvargo) [inline packer/builds/scheduling-builds.html.md]: "optionally automatically" reads a bit strange. Could we say something like "Atlas provides an option to automatically run a Packer build ..."?

File: packer/builds/scheduling-builds.html.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+or to build nightly style images for staging or development environments.
+
+## Enabling Periodic Builds
+
+To enable periodic builds for a build, visit the build settings page in
+Atlas and select the desired interval and click the save button to
+persist the changes. An initial build may immediately run, depending
+on the history, and then will automatically build at the specified interval.
+
+If you have manually run a build separately, Atlas will not queue a new
```
Comment (sethvargo) [inline packer/builds/scheduling-builds.html.md]: "If another build has been run (either manually or via GitHub ingress), Atlas will not queue a build..."

**Comment by sethvargo** [comment]
There are some sentences that I had to re-read because of word groupings, but the technical content looks good :+1:

**Comment by mootpt** [comment]
LGTM

**Pull request merged**